### ISSUE TITLE
Landing page template is missing H1

### ISSUE THREAD

**Issue by nschonni** [uswds/uswds-site#451]
Looks like the other layouts have an H1 heading, but the landing layout was missed.
Maybe the hero callout should be used https://github.com/18F/web-design-standards-docs/blob/0564febab45eed87117808115a094dcbb213de99/_layouts/landing.html#L11

**Comment by nschonni** [comment]
@maya I guess this is also an issue to be addressed in https://github.com/18F/web-design-standards/blob/de246aef2df86ebd8f0f15ebe0eef9f0e5d5fe61/src/components/layouts/landing.njk

**Comment by maya** [comment]
Landing pages can be tricky bc in some cases the logo may be used as the `<h1>`, but it may be simpler to just make the hero title the `<h1>` in this case. Thanks again.

**Comment by maya** [comment]
Moved to this repo: https://github.com/uswds/uswds/issues/2594.
